railties/lib/rails/api/task.rb

```diff
@@ -9,14 +10,21 @@ class Task < RDoc::Task
             README.rdoc
             lib/active_support/**/*.rb
           ),
-          exclude: "lib/active_support/vendor/*"
+          exclude: [
+            'lib/active_support/i18n_railtie.rb',
+            'lib/active_support/xml_mini/*'
+          ]
         },
 
         "activerecord" => {
           include: %w(
             README.rdoc
             lib/active_record/**/*.rb
-          )
+          ),
+          exclude: [
+            'lib/active_record/tasks/*',
```

Comment:
Actually the `ActiveRecord::Tasks` module is `:nodoc`'d everywhere so I guess it's safe to remove them. :blush: